<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_notes</class>
 <widget class="QMainWindow" name="Page_notes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QFrame" name="frameAdditionalPanel">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackedWidget_AdditionalPanel">
          <property name="toolTip">
           <string/>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="pageTable">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tableView">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <attribute name="verticalHeaderShowSortIndicator" stdset="0">
               <bool>true</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitterPageNote_Right">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QFrame" name="frameCurrentNote">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="title">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TextEditor" name="editor"/>
         </item>
        </layout>
       </widget>
       <widget class="AttachPanel" name="attach" native="true"/>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>splitterMain</zorder>
   <zorder>tableView</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEditor</class>
   <extends>QTextEdit</extends>
   <header>Note/texteditor.h</header>
  </customwidget>
  <customwidget>
   <class>AttachPanel</class>
   <extends>QWidget</extends>
   <header>Note/AttachPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
